go.mod

```diff
@@ -0,0 +1,57 @@
+module github.com/cosmos/cosmos-sdk
+
+require (
+	bou.ke/monkey v1.0.1 // indirect
```

Comment:
The only risk I see here is that we can have a mismatch between versions here and gopkg.toml if this PR gets old wrt to develop.
Similarly wrt to branches. git will not help when merging, etc.
I would say we should add a manual step in each PR to confirm that we don't make mistakes when mixing old gopkg and new go.mod files.

Reply:
No worries, I'm constantly rebasing :)